plugins/outputs/newrelic/newrelic.go

```diff
@@ -51,14 +57,16 @@ func (nr *NewRelic) Connect() error {
 	if nr.InsightsKey == "" {
 		return fmt.Errorf("InsightKey is a required for newrelic")
 	}
+	//TODO init nr.client here
```

Comment:
Well, please do it in this PR then. What should this line tell me? :-)

Reply:
Removed